src/main/java/com/kosherjava/zmanim/hebrewcalendar/JewishCalendar.java

```diff
@@ -933,6 +974,58 @@ public Daf getDafYomiYerushalmi() {
 	}
 
 
+	public boolean mashivHaruachStarts() {
+		return getJewishMonth() == 7 && getJewishDayOfMonth() == 22;
+	}
+
+	public boolean mashivHaruachEnds() {
+		return getJewishMonth() == 1 && getJewishDayOfMonth() == 15;
+	}
+
+	public boolean mashivHaruach() {
```

Comment:
@plonibarploni , please rename to isMashivHaruach()

Reply:
Done